core/src/sumeragi/main_loop.rs

```diff
@@ -85,6 +87,16 @@ impl Sumeragi {
         if peer == &self.peer_id {
             return;
         }
+        if matches!(packet, MessagePacket::NormalMessage(_)) {
```

Comment:
So now for every message we send additional message with view_change_proofs? Why do we need to do that?

Reply:
Also this `if` introduces one more *exceptional* behaviour... Can it be refactored with some trait implementation? For example default implemetation for all and specific for the internal type of `NormalMessage` variant?

Reply:
has been removed